### PR TITLE
Ah debian upgrade

### PR DESCRIPTION
**Pull request by ahaessly**
upgrade the version of debian leo uses from jessie to stretch.
this is needed to support the R kernel

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: jupyter-docker/Dockerfile [hunk 1]
```diff
@@ -11,13 +11,14 @@ USER root
...
 ENV DEBIAN_REPO http://cdn-fastly.deb.debian.org
 
-RUN echo "deb $DEBIAN_REPO/debian jessie main\ndeb $DEBIAN_REPO/debian-security jessie/updates main" > /etc/apt/sources.list \
+RUN echo "deb $DEBIAN_REPO/debian stretch main\ndeb $DEBIAN_REPO/debian-security stretch/updates main" > /etc/apt/sources.list \
  && apt-get update && apt-get -yq dist-upgrade \
  && apt-get install -yq --no-install-recommends \
     nano \
+    wget \
+    gnupg \
     #note ca-certificates has to be before wget and curl
```
Comment (rtitle) [inline jupyter-docker/Dockerfile]: Minor, remove this comment if it's not true :)

**Comment by rtitle** [comment]
Oh -- don't trust the automation tests attached to this PR! They use the `dev` docker image by default. To test this change, you need to make a firecloud-develop branch with the same name and update the leonardo-notebooks docker tag to the branch name.

File: jupyter-docker/Dockerfile (hunk 1 above)
Comment (ahaessly) [reply jupyter-docker/Dockerfile]: ah thanks

**Comment by ahaessly** [comment]
jenkins retest

**Pull request merged**